src/olympia/legacy_api/views.py

```diff
@@ -36,7 +36,7 @@
 
 
 ERROR = 'error'
-OUT_OF_DATE = ugettext_lazy(
+OUT_OF_DATE = _(
```

Comment:
I disagree, it's being translated on run-time when we call `OUT_OF_DATE.format(…)` so being lazy here is perfectly fine (and correct).